lib/chef_zero/endpoints/actor_endpoint.rb

```diff
@@ -65,7 +65,13 @@ def put(request)
           else
             response = FFI_Yajl::Parser.parse(result[2], :create_additions => false)
           end
-          response['private_key'] = private_key if private_key
+
+          if request.rest_path[2] == 'clients'
```

Comment:
This is super confusing branching logic - I would expect to see `actor_endpoint_spec.rb` updated.  Unless all the testing is through pedant?